cloudify_aws/ec2/resources/ebs.py

```diff
@@ -82,6 +84,23 @@ def status(self):
         return self.properties[VOLUME_STATE]\
             if self.properties and self.properties.get(VOLUME_STATE) else None
 
+    def attach(self, params):
+
+        return self.make_client_call('attach_volume', params)
+
+    def detach(self, params=None):
```

Comment:
maybe we should add `params = params or {}` to avoid a TypeError on `**params`?

Reply:
Done